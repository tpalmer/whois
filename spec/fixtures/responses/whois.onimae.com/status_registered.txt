GMO Internet, Inc. whois server 2.0

This server only serves information on GMO Internet,Inc.-maintained domain names
For more information, please contact hostmaster@gmo.jp.

Domain Handle: 2564
Domain Name: onamae.com
Created On: 1999-12-17 15:37:03.0
Last Updated On: 2011-12-17 00:15:28.0
Expiration Date: 2012-12-17 06:37:02.0
Status: ACTIVE
Registrant Name: GMO Internet, Inc.
Registrant Organization: GMO Internet, Inc.
Registrant Street1: 26-1 Sakuragaoka - Cerulean Tower
Registrant Street2:
Registrant City: Shibuya-ku
Registrant State: Tokyo
Registrant Postal Code: 150-8512
Registrant Country: JP
Registrant Phone: 0354562555
Registrant Fax:
Registrant Email: admin@onamae.com
Admin Name: GMO Internet, Inc.
Admin Organization: GMO Internet, Inc.
Admin Street1: Cerulean Tower
Admin Street2: 26-1 Sakuragaoka-cho
Admin City: Shibuya-ku
Admin State: Tokyo
Admin Postal Code: 150-8512
Admin Country: JP
Admin Phone: 03-5456-2555
Admin Fax: 03-5456-2556
Admin Email: admin@onamae.com
Billing Name: GMO Internet, Inc.
Billing Organization: GMO Internet, Inc.
Billing Street1: Cerulean Tower
Billing Street2: 26-1 Sakuragaoka-cho
Billing City: Shibuya-ku
Billing State: Tokyo
Billing Postal Code: 150-8512
Billing Country: JP
Billing Phone: 03-5456-2555
Billing Fax: 03-5456-2556
Billing Email: admin@onamae.com
Tech Name: GMO Internet, Inc.
Tech Organization: GMO Internet, Inc.
Tech Street1: Cerulean Tower
Tech Street2: 26-1 Sakuragaoka-cho
Tech City: Shibuya-ku
Tech State: Tokyo
Tech Postal Code: 150-8512
Tech Country: JP
Tech Phone: 03-5456-2555
Tech Fax: 03-5456-2556
Tech Email: admin@onamae.com
Name Server: dns.interq.or.jp
Name Server: dns1.interq.or.jp

