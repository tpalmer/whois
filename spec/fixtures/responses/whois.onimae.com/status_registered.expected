#status
  should: %s == :registered

#available?
  should: %s == false

#registered?
  should: %s == true


#created_on
  should: %s CLASS(time)
  should: %s == Time.parse("1999-12-17 15:37:03.0")

#updated_on
  should: %s CLASS(time)
  should: %s == Time.parse("2011-12-17 00:15:28.0")

#expires_on
  should: %s CLASS(time)
  should: %s == Time.parse("2012-12-17 06:37:02.0")


#registrar
  should: %s CLASS(registrar)
  should: %s.id           == nil
  should: %s.name         == "GMO Internet"
  should: %s.organization == "GMO Internet, Inc."
  should: %s.url          == "http://onimae.com"

#registrant_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type          == Whois::Record::Contact::TYPE_REGISTRANT
  should: %s[0].id            == nil
  should: %s[0].name          == "GMO Internet, Inc."
  should: %s[0].organization  == "GMO Internet, Inc."
  should: %s[0].address       == "26-1 Sakuragaoka - Cerulean Tower"
  should: %s[0].city          == "Shibuya-ku"
  should: %s[0].zip           == "150-8512"
  should: %s[0].state         == "Tokyo"
  should: %s[0].country_code  == "JP"
  should: %s[0].phone         == "0354562555"
  should: %s[0].fax           == nil
  should: %s[0].email         == "admin@onamae.com"

#admin_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type          == Whois::Record::Contact::TYPE_ADMIN
  should: %s[0].id            == nil
  should: %s[0].name          == "GMO Internet, Inc."
  should: %s[0].organization  == "GMO Internet, Inc."
  should: %s[0].address       == "Cerulean Tower\n26-1 Sakuragaoka-cho"
  should: %s[0].city          == "Shibuya-ku"
  should: %s[0].zip           == "150-8512"
  should: %s[0].state         == "Tokyo"
  should: %s[0].country_code  == "JP"
  should: %s[0].phone         == "03-5456-2555"
  should: %s[0].fax           == "03-5456-2556"
  should: %s[0].email         == "admin@onamae.com"

#technical_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type          == Whois::Record::Contact::TYPE_TECHNICAL
  should: %s[0].id            == nil
  should: %s[0].name          == "GMO Internet, Inc."
  should: %s[0].organization  == "GMO Internet, Inc."
  should: %s[0].address       == "Cerulean Tower\n26-1 Sakuragaoka-cho"
  should: %s[0].city          == "Shibuya-ku"
  should: %s[0].zip           == "150-8512"
  should: %s[0].state         == "Tokyo"
  should: %s[0].country_code  == "JP"
  should: %s[0].phone         == "03-5456-2555"
  should: %s[0].fax           == "03-5456-2556"
  should: %s[0].email         == "admin@onamae.com"


#nameservers
  should: %s CLASS(array)
  should: %s SIZE(2)
  should: %s[0] CLASS(nameserver)
  should: %s[0].name == "dns.interq.or.jp"
  should: %s[1] CLASS(nameserver)
  should: %s[1].name == "dns1.interq.or.jp"

